ray-operator/controllers/ray/suite_test.go

```diff
@@ -54,7 +52,7 @@ func TestAPIs(t *testing.T) {
 
 	RunSpecsWithDefaultAndCustomReporters(t,
 		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+		[]Reporter{})
```

Comment:
I know that `printer` was deprecated in controller-runtime, but what's the effect of this change? Will they have the same behavior?

Reply:
Fixed by copying from your PR